roxie/ccd/ccdserver.cpp

```diff
@@ -4935,6 +4935,8 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
         unsigned checkInterval = activity.queryContext()->checkInterval();
         if (checkInterval > timeout)
             checkInterval = timeout;
+        if (acknowledgeAllRequests && checkInterval > packetAcknowledgeTimeout)
```

Comment:
I think we discussed this or something very similar before ?
If a query provides a timeout then checkInterval is set to 1/10th of that timeout.
But if no query timeout provided then yes, I think checkInterval should be less than 5000.

Reply:
Perhaps could be simplified to always setting it to packetAcknowledgeTimeout if acknowledgeAllRequests is true ?